app/src/main/res/drawable/ic_filter_30dp.xml

```diff
@@ -0,0 +1,4 @@
+<vector android:height="30dp" android:viewportHeight="459.0"
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.ic_filter_30dp` appears to be unused
 Unused resources make applications larger and slow down builds.